setup.py

```diff
@@ -44,7 +44,14 @@
         "onnxruntime<1.15.0",
         "transformers>=4.34.0",
     ],
-    "openvino": ["openvino>=2023.2", "onnx", "onnxruntime", "transformers>=4.36.0", "optimum>=1.16.1"],
+    "openvino": [
+        "openvino>=2023.3",
+        "onnx",
+        "onnxruntime",
+        "transformers>=4.36.0",
+        "optimum>=1.16.1",
+        "openvino-tokenizers[transformers]",
```

Comment:
Decided to remove this during our meeting. Let's return it back when `openvino-tokenizers-nightly` is published.

Reply:
Done.